bricks/r13n/hooks/post_gen.dart

```diff
@@ -6,12 +6,14 @@ import 'package:meta/meta.dart';
 
 Future<void> run(HookContext context) async {
   final runProcess = ProcessOverrides.current?.runProcess ?? Process.run;
+  final progress = context.logger.progress('Running "flutter format ."');
   final result = await runProcess(
```

Comment:
Okay okay hear me out.. the progress logger should be done magically in `runProcess`?

Reply:
That could be a cool enhancement but what if you don't want that behavior?

Reply:
In the context of this hook, why wouldn't you?

Reply:
Ah yeah specifically for this hook it makes sense